server/mscalendar/notification_format.go

```diff
@@ -0,0 +1,247 @@
+package mscalendar
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/mattermost/mattermost-plugin-mscalendar/server/config"
+	"github.com/mattermost/mattermost-plugin-mscalendar/server/mscalendar/views"
+	"github.com/mattermost/mattermost-plugin-mscalendar/server/remote"
+	"github.com/mattermost/mattermost-plugin-mscalendar/server/utils/fields"
+
+	"github.com/mattermost/mattermost-server/v6/model"
+)
+
+func (processor *notificationProcessor) newSlackAttachment(n *remote.Notification) *model.SlackAttachment {
```

Comment:
Just wondering, did this file's contents change at all after copying them here?

Reply:
Not sure what you mean. I only moved the parts of the logic that were about formatting, this file didn't exist previously. And I didn't modify any code present here, it's a direct copy & paste.

(Though I will modify some of it to make it similar to the mockups in another PR.)

Reply:
> And I didn't modify any code present here, it's a direct copy & paste.

This was actually what I was wondering. LGTM :+1: